docs/configuration.rst

```diff
@@ -66,6 +66,12 @@ The following environment variables for the tracer are supported:
      description: |
          Enables sending :ref:`telemetry <Instrumentation Telemetry>` events to the agent.
 
+   DD_TELEMETRY_METRICS_ENABLED:
```

Comment:
We don’t need to add this new configuration to our public docs. We can use it internally for tests. 
```suggestion
```

Reply:
:+1: